### PR TITLE
#18 LCOM5

### PR DESCRIPTION
**Pull request by llorllale**
as per #18 

>'LCOM5' is a 1996 revision by B. Henderson-Sellers, L. L. Constantine, and I. M. Graham, of the initial LCOM metric proposed by MIT researchers.

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#115](https://github.com/yegor256/jpeek/issues/115) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#115](https://github.com/yegor256/jpeek/issues/115) is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/test/java/org/jpeek/MetricsTest.java [hunk 1]
```diff
@@ -35,6 +35,12 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
+// @todo #18:30min Impediment: #103 must be fixed before LCOM5 is tested
```
Comment (amihaiemil) [inline src/test/java/org/jpeek/MetricsTest.java]: @llorllale Same here:

1) these 2 puzzles in the javadoc

2) As in the other PR, shouldn't it return something else than ``NaN`` until 103 is solved? Maybe -1?

File: src/main/resources/org/jpeek/metrics/LCOM5.xsl [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+SOFTWARE.
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="2.0">
+  <xsl:param name="ctors" select="0"/>
+  <xsl:template match="skeleton">
+    <metric>
+      <xsl:apply-templates select="@*"/>
+      <title>LCOM5</title>
+      <description>
+        <xsl:text>'LCOM5' is a 1996 revision by B. Henderson-Sellers,
```
Comment (amihaiemil) [inline src/main/resources/org/jpeek/metrics/LCOM5.xsl]: @llorllale The first paragraph of this, in the PRs description please :)

Comment (llorllale) [reply src/main/resources/org/jpeek/metrics/LCOM5.xsl]: @amihaiemil done

File: src/test/java/org/jpeek/MetricsTest.java (hunk 1 above)
Comment (llorllale) [reply src/test/java/org/jpeek/MetricsTest.java]: @amihaiemil my replies to these two concerns in the other PR (#113) apply here. Please let me know.

Comment (amihaiemil) [reply src/test/java/org/jpeek/MetricsTest.java]: @llorllale ok

**Comment by amihaiemil** [comment]
@rultor merge it

**Comment by yegor256** [comment]
@rultor merge

**Comment by amihaiemil** [comment]
@llorllale please have a look, one of the puzzles cannot be parsed, that's why the merge won't work.

**Comment by amihaiemil** [comment]
@llorllale ping

**Comment by llorllale** [comment]
@amihaiemil fixed

**Comment by amihaiemil** [comment]
@yegor256 please try to merge now

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was successfully finished: +15 points just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), total is [+195](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+8800](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
This pull request [#115](https://github.com/yegor256/jpeek/issues/115) is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by amihaiemil** [comment]
@yegor256 0crat is at it again, reassigned me here.

It didn't pay me this time at all, only gave me points, maybe it has something to do with it :)

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) resigned from [#115](https://github.com/yegor256/jpeek/issues/115), since the job is not in scope anymore

**Comment by 0crat** [comment]
The job [#115](https://github.com/yegor256/jpeek/issues/115) is now out of scope

**Comment by 0crat** [comment]
Order was successfully finished: +30 points just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), total is [+225](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+8875](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#115](https://github.com/yegor256/jpeek/issues/115) is now out of scope

**Pull request merged**